src/js/components/Poll/PublicRegisterModal.vue

```diff
@@ -190,13 +190,13 @@ export default {
 
 			if (!this.isValidName) {
 				return {
-					result: t('polls', '{username} is invalid or reserved.', { username: this.userName }),
+					result: t('polls', 'Name {username} is invalid or reserved.', { username: this.userName }),
```

Comment:
"Name Peter is invalid" sounds strange to me. Either it needs an article ("The name Peter is invalid") or the term stays as before ("Peter is invalid").

Reply:
"User name **Peter** is invalid" would be my suggestion.

Reply:
@rakekniven We had feedback, that the term `username` could be misleading for users who visit a public poll and are also a registered user of this site. This was the initial reason, why the term `username` was removed.

Reply:
Ok, good catch.

So what about "Participant name **Peter** is invalid or reserved"?